tests/test_api.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+
+try:
+    from .test_genius import genius
+except ModuleNotFoundError:
+    from test_genius import genius
+
+
+class TestAPI(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        print("\n---------------------\nSetting up API tests...\n")
+
+    def test_account(self):
+        msg = ("No user detail was returned. "
+               "Are you sure you're using a user access token?")
+        r = genius.account()
+        self.assertTrue("user" in r, msg)
+
+    def test_annotation(self):
+        msg = "Returned annotation API path is different than expected."
+        id_ = 10225840
+        r = genius.annotation(id_)
+        real = r['annotation']['api_path']
+        expected = '/annotations/10225840'
+        self.assertEqual(real, expected, msg)
+
+    def test_manage_annotation(self):
+        example_text = 'The annotation'
+        new_annotation = genius.create_annotation(
+            example_text,
+            'https://example.com',
+            'illustrative examples',
+            title='test')['annotation']
+        msg = 'Annotation text did not match the one that was passed.'
+        self.assertEqual(new_annotation['body']['plain'], example_text, msg)
+
+        example_text_two = 'Updated annotation'
+        r = genius.update_annotation(
+            new_annotation['id'],
+            example_text_two,
+            'https://example.com',
+            'illustrative examples',
+            title='test'
+        )['annotation']
+        msg = 'Updated annotation text did not match the one that was passed.'
+        self.assertEqual(r['body']['plain'], example_text_two, msg)
+
+        r = genius.upvote_annotation(11828417)
+        msg = 'Upvote was not registered.'
+        self.assertTrue(r is not None, msg)
+
+        r = genius.downvote_annotation(11828417)
+        msg = 'Downvote was not registered.'
+        self.assertTrue(r is not None, msg)
+
+        r = genius.unvote_annotation(11828417)
+        msg = 'Vote was not removed.'
+        self.assertTrue(r is not None, msg)
+
+        msg = 'Annotation was not deleted.'
+        r = genius.delete_annotation(new_annotation['id'])
+        self.assertEqual(r, 204, msg)
```

Comment:
I get a 403 Client Error on this test. Does it run successfully for you?

```
403 Client Error: Forbidden for url: https://api.genius.com/annotations?text_format=plain
403 Forbidden
```

Reply:
I just ran this test and it ran successfully. My comment above will probably solve this too.

Reply:
Yep, the test passes for me now.